app/models/preferences.js

```diff
@@ -0,0 +1,11 @@
+module.exports = (sequelize, DataTypes) => {
+  return sequelize.define('preferences', {
```

Comment:
This should be `preference` rather than `preferences` as needs to match what follows `db.` in `routes/preferences`

We could debate over whether these should be single or plural but my view is they represent a model rather than a collection so singular makes sense.